src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -118,12 +118,18 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('identity', help="Use assigned managed identity resource id or name if in the same resource group")
         c.argument('key_encryption_key', help="Key vault key uri. To enable automated rotation, provide a version-less key uri. For manual rotation, provide a versioned key uri.")
 
+    with self.argument_context('acr create') as c:
+        c.argument('metadata_search', arg_type=get_enum_type(['Enabled', 'Disabled']), is_preview=True, help="Enable or disable the metadata-search feature for the registry. If not specified, this is set to disabled by default.")
```

Comment:
```suggestion
        c.argument('enable_metadata_search', arg_type=get_three_state_flag(), is_preview=True, help="Enable or disable the metadata-search feature for the registry. If not specified, this is set to disabled by default.")
```

Reply:
>  For now, I make it as: --metadata-search Enabled/Disabled. Is it a good practice? Another pattern I saw is using it like a boolean flag, such as: --metadata-search-enabled.

Yes, we recommend that you follow the CLI unified specification for bool type parameters [doc link](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags)
This way, you can also keep the consistency with the other bool type parameters of ACR commands to avoid confusing customers